src/web/lib/components/ThemeCustomBackgroundPicker/index.js

```diff
@@ -156,23 +162,43 @@ const BackgroundList = SortableContainer(props => {
     themeCustomBackgrounds,
     themeCustomImages
   } = props;
+
+  const getImage = (name) => {
+    let image = storedImages.get(name);
+
+    if (image) {
+      // eslint-disable-next-line no-unused-vars
+      const { importing, ...rest } = image;
+
+      const newImage = {
+        ...rest
+      };
+      // add the image back into local storage.
```

Comment:
When would this happen?

I'm fine with automatically adding the image back upon redo/undo, but then when the user use redo/undo again, then the image should be deleted again.

Reply:
this adds the image if its looking for it on undo/redo and the image is missing.

ok thanks for the feedback. I'll look into this..